<launch>
  <param name="move_group_name" type="string" value="arm_er7" />
  <node name="rtp_gui_node" pkg="rtp_gui" type="rtp_gui_node" output="screen" />
  <node name="rtp_node" pkg="rtp_gui" type="rtp_node" output="screen" />

</launch>
